zerver/webhooks/gogs/tests.py

```diff
@@ -9,6 +11,14 @@ class GogsHookTests(WebhookTestCase):
     URL_TEMPLATE = "/api/v1/external/gogs?&api_key={api_key}"
     FIXTURE_DIR_NAME = 'gogs'
 
+    def build_webhook_url(self, branches=None):
```

Comment:
Maybe it'd be worth to extend method from `WebhookTestCase` by adding `**kwargs`?